feat(unity): IL2CPP line number limitations (#7629)
getsentry/sentry-docs: src/platforms/unity/configuration/il2cpp.mdx

```

The support for line numbers in C# Exceptions with IL2CPP relies on the debug symbol upload. With the automated debug symbol upload enabled, the [Sentry SDK will upload the line mapping automatically](/platforms/unity/native-support/#debug-symbols).

### Known Limitations

> Currently, line number support is limited when programmatically subscribing to button click events via `Button.onClick.AddListener`. This can be worked around by setting up callbacks through the editor.

Reviewer: ```suggestion
Currently, line number support is limited when programmatically subscribing to button click events via `Button.onClick.AddListener`. However, you can set up callbacks through the editor to work around this.
```
